XMM/analysis-xmm-short-intro.md

```diff
@@ -0,0 +1,168 @@
+---
+jupyter:
+  jupytext:
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.16.0
+  kernelspec:
+    display_name: (heasoft)
```

Comment:
Please change to start in the xmmsas environment by default. All the other notebooks are ok except this one.

Reply:
Fixed.